openvdb_points_houdini/houdini/SOP_NodeVDBPoints.h

```diff
@@ -279,14 +280,23 @@ class OPENVDB_HOUDINI_API SOP_NodeVDBPoints: public SOP_NodeVDB
                 }
                 const openvdb::NamePair& type = descriptor->type(it->second);
 
+                openvdb::Name valueType = type.first;
+                openvdb::Name codecType = type.second;
+
+                // modify string attribute types to use "str"
+
+                if (array.isString()) {
+                    valueType = "str";
+                }
```

Comment:
It worries me that we have to check a variety of flags after querying the value type to be 100% certain of the actual type of the array.